pydantic_ai_slim/pydantic_ai/ext/aci_tools.py

```diff
@@ -0,0 +1,82 @@
+from aci import ACI
+
+def clean_schema(schema):
```

Comment:
Please add a `_` underscore to make this a private function, as we only want to use this inside this file and don't want Pydantic AI users to use it directly.